toolkit/utilitynetworks/src/main/java/com/arcgismaps/toolkit/utilitynetworks/ui/TraceOptionsScreen.kt

```diff
@@ -328,15 +345,23 @@ internal fun AdvancedOptions(
         Column {
             if (showName) {
                 val focusManager = LocalFocusManager.current
-                AdvancedOptionsRow(name = stringResource(id = R.string.name)) {
-                    var text by rememberSaveable { mutableStateOf("test trace result name") }
+                Row(
+                    modifier = modifier
+                        .fillMaxWidth()
+                        .height(80.dp)
+                        .padding(10.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.SpaceBetween
+                ) {
+                    var text by rememberSaveable { mutableStateOf(defaultTraceName) }
                     TextField(
```

Comment:
Probably subjective, but I find the `OutlinedTextField` is a nicer looking text field for user inputs: 
```suggestion
                    OutlinedTextField(
```